main.js

```diff
@@ -781,7 +781,16 @@ function sendMessage(text, user, chatId, options) {
         text = (text || '').toString();
         text = text.replace('@' + m[1], '').trim().replace(/\s\s/g, ' ');
         const re = new RegExp(m[1], 'i');
-        const id = users.find(id => (!adapter.config.useUsername && users[id].firstName.match(re)) || (adapter.config.useUsername && users[id].userName.match(re)));
+        let id = "";
+        for (const id_t in users) {
+            if (!users.hasOwnProperty(id_t)) {
+                continue;
+            }
+            if ((adapter.config.useUsername && users[id_t].userName.match(re)) || (!adapter.config.useUsername && users[id_t].firstName.match(re))) {
```

Comment:
But this is the same like:

```
         const id = users.find(id => ((adapter.config.useUsername && users[id_t].userName.match(re)) || (!adapter.config.useUsername && users[id_t].firstName.match(re)));
```

Reply:
No, because "users" object has no .find method. I attach original problem to issue. My instance can't send messages to specified user with this error:
```
error	at processImmediate (internal/timers.js:456:21)
error	at Immediate.<anonymous> (/opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:5388:34)
error	at Adapter.emit (events.js:310:20)
error	at Adapter.<anonymous> (/opt/iobroker/node_modules/iobroker.telegram/main.js:241:21)
error	at sendMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:785:26)
error	(23509) TypeError: users.find is not a function
error	(23509) uncaught exception: users.find is not a function
```

Reply:
aahhh then users is no array ... i see